File: src/Library/Client/SoundbridgeClientException.vb
Imports System.Runtime.Serialization

<Serializable()> _
Public Class SoundbridgeClientException
    Inherits System.Exception

    Public Sub New()

    End Sub

    Public Sub New(ByVal message As String)
        MyBase.New(message)
    End Sub

    Public Sub New(ByVal message As String, ByVal innerException As Exception)
        MyBase.New(message, innerException)
    End Sub

    Protected Sub New(ByVal info As SerializationInfo, ByVal context As StreamingContext)
        MyBase.New(info, context)
    End Sub
End Class

File: src/Pixa.Soundbridge/Client/SoundbridgeClientException.cs
using System;
using System.Runtime.Serialization;

namespace Pixa.Soundbridge.Client
{
    [Serializable()]
    public class SoundbridgeClientException : Exception
    {
        public SoundbridgeClientException()
        {
        }

        public SoundbridgeClientException(string message) : base(message)
        {
        }

        public SoundbridgeClientException(string message, Exception innerException) : base(message, innerException)
        {
        }

        protected SoundbridgeClientException(SerializationInfo info, StreamingContext context) : base(info, context)
        {
        }
    }
}